google/cloud/pubsub/internal/subscription_session.h

```diff
@@ -34,10 +35,28 @@ namespace cloud {
 namespace pubsub_internal {
 GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 
+// TODO(#9327) - move this to application_callback.h
+/**
+ * Defines the interface for application-level callbacks with exactly-once
+ * delivery.
+ *
+ * Applications provide a callable compatible with this type to receive
+ * messages.  They acknowledge (or reject) messages using
+ * `ExactlyOnceAckHandler`. This is a move-only type to support asynchronously
```

Comment:
nit: s/asynchronously/asynchronous/

Reply:
also super nit: earlier in the paragraph you put two spaces after the period, but here you put one.

Reply:
Fixed, thanks